src/components/DetailsOverlay/SearchRangeInput/index.tsx

```diff
@@ -0,0 +1,72 @@
+import styled from "@emotion/styled";
+import TextInput from "@leafygreen-ui/text-input";
+import { QueryParams } from "constants/queryParams";
+import { size } from "constants/tokens";
+import { useQueryParam } from "hooks/useQueryParam";
+import { FilterRow, StyledSubtitle } from "../styles";
+
+const SearchRangeInput: React.FC = () => {
+  const [lowerBound, setLowerQueryParam] = useQueryParam<number | undefined>(
+    QueryParams.LowerRange,
+    undefined
+  );
+  const [upperBound, setUpperBound] = useQueryParam<number | undefined>(
+    QueryParams.UpperRange,
+    undefined
+  );
+
+  const updateLowerBound = (newVal: string) => {
+    if (newVal === "") {
+      setLowerQueryParam(undefined);
+    } else {
+      setLowerQueryParam(parseInt(newVal, 10));
+    }
+  };
+
+  const updateUpperBound = (newVal: string) => {
+    if (newVal === "") {
+      setUpperBound(undefined);
+    } else {
+      setUpperBound(parseInt(newVal, 10));
+    }
```

Comment:
I think setting this to -1 should remove the upperbound

Reply:
Hmm when I do that it adds `upper=-1` to the URL. It could be best to just remove it from the URL if it's not a valid upper bound

Reply:
I think this just needs a conditional something like...
```ts
// replace this with a better name :) 
const newValNumber = parseInt(newVal,10);
if (newValNumber === -1){
   setUpperBound(undefined);
} else{
   setUpperBound(newValNumber);
}
```